rebasehelper/cli.py

```diff
@@ -249,10 +268,11 @@ def run():
                 raise RebaseHelperError('Wrong format of --builder-options. It must be in the following form:'
                                         ' --builder-options="--desired-builder-option".')
             cli = CLI()
-            execution_dir, results_dir, debug_log_file, report_log_file = Application.setup(cli)
+            execution_dir, results_dir, debug_log_file = Application.setup(cli)
             if not cli.verbose:
                 handler.setLevel(logging.INFO)
-            app = Application(cli, execution_dir, results_dir, debug_log_file, report_log_file)
+            app = Application(cli, execution_dir, results_dir, debug_log_file)
+            app.conf.colors = CliHelper.set_colors(app)
```

Comment:
You definitely shouldn't redefine the attribute. You should use the method in some `print` or rather `sys.stdout.write` wrapper.